lightnode/kvstore/store.go

```diff
@@ -0,0 +1,41 @@
+package kvstore
+
+import (
+	"errors"
+	"github.com/syndtr/goleveldb/leveldb/iterator"
+)
+
+// Store implements a key-value store. May be backed by a database like LevelDB.
+type Store interface {
```

Comment:
Comment the thread safety of these interfaces

Reply:
Commend changed to the following:

```
// Implementations of this interface are expected to be thread-safe.
```